manifests/activeresponse.pp

```diff
@@ -1,20 +1,30 @@
 # Wazuh App Copyright (C) 2019 Wazuh Inc. (License GPLv2)
 #Define for a specific ossec active-response
 define wazuh::activeresponse(
-  $command_name,
-  $ar_location           = 'local',
-  $ar_level              = 7,
-  $ar_agent_id           = '',
-  $ar_rules_id           = [],
-  $ar_timeout            = 300,
-  $ar_repeated_offenders = '',
+  $active_response_name               = 'Rednering active-response template',
```

Comment:
Re**n**dering active-response template

Reply:
Fixed in. https://github.com/wazuh/wazuh-puppet/pull/234/commits/701a7e45f8999b127b440c425eabafd3cda50f8c